packages/data-broker/src/services/dhis/DhisService.js

```diff
@@ -164,12 +170,17 @@ export class DhisService extends Service {
   }
 
   async pullMetadata(dataSources, type, options) {
-    const { organisationUnitCode: entityCode, dataServices = DEFAULT_DATA_SERVICES } = options;
+    const { organisationUnitCodes: entityCodes, dataServices = DEFAULT_DATA_SERVICES } = options;
```

Comment:
Do we still use `organisationUnitCode` anywhere? If so should this for backwards compatability:
```suggestion
    const { organisationUnitCode: entityCode, organisationUnitCodes: entityCodes, dataServices = DEFAULT_DATA_SERVICES } = options;
    const dataServiceEntityCode = entityCode || entityCodes[0];
```